### ISSUE TITLE
Missing completions for whitebox macros

### ISSUE THREAD

**Issue by soronpo** [scalameta/metals#2195]
**Describe the bug**
In this very specific situation Metals fails to get completions of a whitebox macro fields.
Notice the line `new java.io.File(c.enclosingPosition.source.path).getParentFile.getCanonicalPath` is the one causing the issue.

The complete build is available https://github.com/soronpo/bad_reflection
**MacroLand.scala**
```scala
object MacroLand {
  import scala.reflect.macros.whitebox
  def apply() : Any = macro applyMacro
  def applyMacro(c: whitebox.Context)() : c.Tree = {
    import c.universe._
    new java.io.File(c.enclosingPosition.source.path).getParentFile.getCanonicalPath //Without this line, completions are working
    q"""
    object Foo {
      case class SomeClass() {
        val someField = 1
      }
    }
    Foo
    """
  }
}
```

**Test.scala**
```scala
object Test extends App {
  val m = MacroLand()
  import m.S //Try complete me here
  import m.SomeClass //error highlighting is OK, but no completions are offered
}
```

**To Reproduce**

1. Use the following repo: https://github.com/soronpo/bad_reflection
2. In the file `bad_reflection\core\src\main\scala\Test.scala` try to do import completions for `m`

**Expected behavior**

Completions should work.

**Installation:**

- Operating system: Windows
- Editor: Visual Studio Code
- Metals version: v0.9.4

**Search terms**

Whitebox macros

**Comment by tgodzik** [comment]
Thanks for reporting! I seem to be able to autocomplete it:

![bad_reflection](https://user-images.githubusercontent.com/3807253/98974600-62610b00-2515-11eb-81c4-2319ae1bd503.gif)

Might be a weird issue with zinc :thinking: 

Some of the issues might be resolved by upgrading zinc in Bloop, but it might take us a while.

**Comment by soronpo** [comment]
@tgodzik I retested it now, and it works for me as well. It might have been the latest release update that fixed this (it was tested with 0.9.4). Thanks!
